src/mangata.ts

```diff
@@ -0,0 +1,272 @@
+      getChain: async () => await getChain(instancePromise),
+      waitForNewBlock: async (blockNumber?: number) =>
+        await waitForNewBlock(instancePromise, blockNumber)
+    },
+    tokens: {
+      transferAllTokens: async (args: Transfer) =>
+        await transferAllTokens(instancePromise, args, false),
+      transferTokens: async (args: TransferTokens) =>
+        await transferTokens(instancePromise, args, false)
+    },
+    submitableExtrinsic: {
+      createPool: async (args: CreatePool) =>
+        await createPool(instancePromise, args, true),
+      claimRewards: async (args: Liquidity) =>
+        await claimRewards(instancePromise, args, true),
+      sellAsset: async (args: SellAsset) =>
+        await sellAsset(instancePromise, args, true),
+      buyAsset: async (args: BuyAsset) =>
+        await buyAsset(instancePromise, args, true),
+      mintLiquidity: async (args: MintLiquidity) =>
+        await mintLiquidity(instancePromise, args, true),
+      burnLiquidity: async (args: BurnLiquidity) =>
+        await burnLiquidity(instancePromise, args, true),
+      activateLiquidity: async (args: Liquidity) =>
+        await activateLiquidity(instancePromise, args, true),
+      deactivateLiquidity: async (args: Liquidity) =>
+        await deactivateLiquidity(instancePromise, args, true),
+      transferAllTokens: async (args: Transfer) =>
+        await transferAllTokens(instancePromise, args, true),
+      transferTokens: async (args: Transfer & { amount: TokenAmount }) =>
+        await transferTokens(instancePromise, args, true)
+    },
+    query: {
+      getNonce: async (address: Address) =>
+        await getNonce(instancePromise, address),
+      getLiquidityTokenId: async (
+        firstTokenId: TokenId,
+        secondTokenId: TokenId
+      ) =>
+        await getLiquidityTokenId(instancePromise, firstTokenId, secondTokenId),
+      getTotalIssuance: async (tokenId: TokenId) =>
+        await getTotalIssuance(instancePromise, tokenId),
+      getTokenBalance: async (address: Address, tokenId: TokenId) =>
+        await getTokenBalance(instancePromise, address, tokenId),
+      getTokenInfo: async (tokenId: TokenId) =>
+        await getTokenInfo(instancePromise, tokenId),
+      getLiquidityTokenIds: async () =>
+        await getLiquidityTokenIds(instancePromise),
+      getLiquidityTokens: async () => await getLiquidityTokens(instancePromise),
+      getBlockNumber: async () => await getBlockNumber(instancePromise),
+      getOwnedTokens: async (address: Address) =>
+        await getOwnedTokens(instancePromise, address),
+      getAssetsInfo: async () => await getAssetsInfo(instancePromise),
+      getInvestedPools: async (address: Address) =>
+        await getInvestedPools(instancePromise, address),
+      getAmountOfTokenIdInPool: async (
+        firstTokenId: TokenId,
+        secondTokenId: TokenId
+      ) =>
+        await getAmountOfTokenIdInPool(
```

Comment:
`getAmountOfTokensInPool` might be better name. Ids as params are explanatory enough